grails-app/i18n/messages.properties

```diff
@@ -2928,7 +2928,7 @@ react.invoice.buyerOrganization.label=Buyer organization
 react.invoice.createdBy.label=Created by
 react.invoice.viewDetails.label=View Invoice Details
 react.invoice.edit.label=Edit Invoice
-react.invoice.error.fetching.label=Could not fetch list of invoices
+react.invoice.error.fetching.label=Could not fetch list of invoices (most probably due to abort while changing location)
```

Comment:
Where did this label come from?

Reply:
obviously from my imagination :( what would you suggest?

Reply:
Why do we need to add more context here? I would just say "Unable to fetch invoices" and leave it at that.

Reply:
Or "Unable to fetch invoices. Try refreshing the page."